src/components/database/create-database/CreateDatabase.jsx

```diff
@@ -47,8 +51,14 @@ const CreateDatabase = (props) => {
   }
 
   const handleDbSubmit = () => {
-    dbEnable(projects, props.projectId, getFieldValue("alias"), getFieldValue("connectionString"), defaultDBRules, selectedDB, (err) => {
-      if(!err) props.handleSubmit()
+    validateFields((err, values) => {
+      if (!err) {
+        dbEnable(projects, props.projectId, getFieldValue("alias"), getFieldValue("connectionString"), defaultDBRules, selectedDB, (err) => {
+          if (!err) {
+            history.push(`/mission-control/projects/${props.projectId}/database/${selectedDB}/overview`)
```

Comment:
This shouldn't be done here. Because this component is even used in create project steps, it would redirect there also. That's why I was taking it as a prop.